dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/LocalHostsSupplier.java

```diff
@@ -16,22 +16,60 @@
 package com.netflix.dynomitemanager.supplier;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
+import org.apache.commons.lang.StringUtils;
+
 import com.google.common.base.Supplier;
+import com.google.inject.Inject;
 import com.netflix.astyanax.connectionpool.Host;
+import com.netflix.dynomitemanager.sidecore.IConfiguration;
 
 public class LocalHostsSupplier implements HostSupplier {
+	
+	private IConfiguration config;
+
+	@Inject
+	public LocalHostsSupplier(IConfiguration config) {
+		this.config = config;
+	}	
+	
      @Override
      public Supplier<List<Host>> getSupplier(String clusterName) {
-       final List<Host> hosts = new ArrayList<Host>();
-          hosts.add(new Host("127.0.0.1", 9160).setRack("localdc"));
-          return new Supplier<List<Host>>() {
-          
-          @Override
-          public List<Host> get() {
-             return hosts;
-          }
-       };
+    	  final List<Host> hosts = new ArrayList<Host>();
+    	  
+    	  String bootCluster = config.getBootClusterName();
+    	  
+    	  if(bootCluster == null)
+    		  bootCluster = "";
+    	  
+    	  if (bootCluster.equals(clusterName)){
+              
+    		  String seeds = System.getenv("DM_CASSANDRA_CLUSTER_SEEDS");
+    		  
+    		  if (seeds == null || "".equals(seeds)) 
+    			  throw new RuntimeException("Cassandra Host Names can not be blank. At least one host is needed. Please use env var: DM_CASSANDRA_CLUSTER_SEEDS.");
+    		  
+    		  List<String> cassHostnames = new ArrayList<String>(Arrays.asList(StringUtils.split(seeds, ",")));
+
+              if(cassHostnames.size() == 0)
```

Comment:
Are we making LocalHostsSupplier depended on Cassandra only? It would be nice abstract it, the same we have done for everything else DM. What do you think?